.github/workflows/update-dependencies.yml

```diff
@@ -0,0 +1,34 @@
+name: Update dependencies
+
+on: [repository_dispatch]
+
+jobs:
+  update-webex-component-adapter-interfaces:
+    runs-on: ubuntu-latest
+    if: github.event.action == 'update-webex-component-adapter-interfaces'
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2-beta
+      - run: |
+          wanted=$(npm outdated @webex/component-adapter-interfaces --json | grep "wanted")
```

Comment:
maybe a simpler version
```sh
npm outdated @webex/components --json | grep wanted | cut -f4 -d\"
1.62.0
```

`npm outdated` has also a `--parseable` option but it separates tokens by colons (:), and on windows the same character is used for the drive name (eg: `C:`)